gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/CseValidHandler.java

```diff
@@ -165,25 +230,20 @@ private void computeTimestampForExportCorner(TTimestampWrapper timestampWrapper,
             BigDecimal miecValue = timestampWrapper.getMibiecValue().subtract(timestampWrapper.getAntcfinalValue());
             tcDocumentTypeWriter.fillTimestampExportCornerSuccess(timestampWrapper.getTimestamp(), miecValue);
         } else {
-            if (isFranceInArea(timestamp)) {
-                runDichotomyForExportCorner(timestampWrapper, cseValidRequest, tcDocumentTypeWriter, true);
-            } else if (isFranceOutArea(timestamp)) {
-                runDichotomyForExportCorner(timestampWrapper, cseValidRequest, tcDocumentTypeWriter, false);
-            } else {
-                throw new CseValidInvalidDataException("France must appear in InArea or OutArea");
+            timestampWrapper.initExportingCountryMap();
+            if (areAllRequiredFilesPresent(timestampWrapper, cseValidRequest, tcDocumentTypeWriter)) {
+                Network network = fileImporter.importNetwork(cseValidRequest.getCgm().getFilename(), cseValidRequest.getCgm().getUrl());
+                double shiftValue = computeShiftValue(timestampWrapper);
+                cseValidNetworkShifter.shiftNetwork(shiftValue, network, timestampWrapper, cseValidRequest.getGlsk().getUrl());
+                if (cseValidRaoValidator.isNetworkSecure(network, cseValidRequest, cseValidRequest.getExportCrac().getUrl())) {
```

Comment:
It is not necessarily export crac here you have to check if france is exporting or not